### PR TITLE
NOISSUE - Fix CI script

### PR DESCRIPTION
**Pull request by dborovcanin**
### What does this do?
This pull request fixes CI script to work with new versions of Go.

### Which issue(s) does this PR fix/relate to?
There is no such issue.

### List any changes that modify/break current functionality
There are no significant changes.

### Have you included tests for your changes?
No.

### Did you document any new/modified functionality?
N/A

### Notes
N/A

### CODE REVIEW

**Review by manuio — DISMISSED** [review]
LGTM

File: scripts/ci.sh [hunk 1]
```diff
@@ -1,39 +1,34 @@
...
 function version_gt() { test "$(printf '%s\n' "$@" | sort -V | head -n 1)" != "$1"; }
 
 update_go() {
     CURRENT_GO_VERSION=$(go version | sed 's/[^0-9.]*\([0-9.]*\).*/\1/')
     if version_gt $GO_VERSION $CURRENT_GO_VERSION; then
         echo "Updating go version from $CURRENT_GO_VERSION to $GO_VERSION ..."
+        # remove other Go version from path
+        sudo rm -rf /usr/bin/go
         sudo rm -rf /usr/local/go
+        sudo rm -rf /usr/local/bin/go
```
Comment (drasko) [inline scripts/ci.sh:17]: Why both `/usr/bin/go` and `/usr/local/bin/go`? Is Semaphore holding `go` in both of these locations?

**Review by drasko — CHANGES_REQUESTED** [review]

**Review by manuio — DISMISSED** [review]
LGTM

File: scripts/ci.sh (hunk 1 above)
Comment (dborovcanin) [reply scripts/ci.sh:17]: I am unsure how Semaphore works, so I removed all the relevant go directories to avoid a potential collision.

**Review by manuio — DISMISSED** [review]

**Review by manuio — APPROVED** [review]
LVGTM

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**